sdk/python/examples/jobset.py

```diff
@@ -0,0 +1,43 @@
+import jobset
+from jobset.models.jobset_v1alpha2_job_set import JobsetV1alpha2JobSet
+from kubernetes.client.models.v1_job_template_spec import V1JobTemplateSpec
+
+jobset_example = jobset.models.jobset_v1alpha2_job_set.JobsetV1alpha2JobSet(
```

Comment:
Can we use defaults for some of these parameters to simplify the spec?  For example We can let success policy be defaulted, and Network can be defaulted, etc.

Reply:
done.